charts/invenio/templates/install-init-job.yaml

```diff
@@ -56,10 +56,24 @@ spec:
             secretKeyRef:
               name: invenio-secrets
               key: INVENIO_CSRF_SECRET_SALT
+        volumeMounts:
+          {{- if .Values.invenio.vocabularies }}
```

Comment:
I think this if is redundant or misplaced. I have not run it myself to verify, but I think `range` will run 0 times (not error) if `.Values.invenio.vocabularies` is null or has no elements.

On the other hand, if `.Values.invenio.vocabularies` is null then you will still have the `volumeMounts` field printed with empty contents (since the `volumeMounts` is defined outside of the `if`).